src/js/plugin/sdk/services/MarathonService.js

```diff
@@ -0,0 +1,23 @@
+
+import config from "../../../config/config";
+import ajaxWrapper from "../../../helpers/ajaxWrapper";
+import Utils from "../utils";
+
+export default class MarathonService {
+
+  static request(opts = {}) {
+    let method = "GET";
+    let resource = Utils.addLeadingSlashIfNedded(opts.resource);
```

Comment:
I not  sure if we need a dedicated util to prepend a slash. Applying a simple `RegExp` like the following one should do the trick.

```js
resource.replace(/\/?(.*)/,"/$1")
```

Reply:
Perfect. Will change to use a simple regex.